src/test/TestLanguage.java

```diff
@@ -0,0 +1,35 @@
+package test;
+
+import view.Painting;
+import model.Canvas;
+import model.presentation.awt.AWTFrame;
+import model.presentation.xml.SVGFile;
+import language.BasicPainter;
+
+/**
+ * This could be the final main class in the project. It read user's drawing task and draw it. Then it shows the
+ * painting in JFrame and in a SVG file.
+ *
+ * Created by Xiaxing SHI on 21/11/15.
+ */
+public class TestLanguage {
```

Comment:
Could you make it a TestClass for now? we will change the actual main eventually after.